homeassistant/components/sensor/sense.py

```diff
@@ -72,34 +90,62 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     password = config.get(CONF_PASSWORD)
 
     data = Senseable(username, password)
+    realtime_data_wrapper = ThrottledRealtimeData(data)
 
     @Throttle(MIN_TIME_BETWEEN_DAILY_UPDATES)
     def update_trends():
         """Update the daily power usage."""
         data.update_trend_data()
 
-    @Throttle(MIN_TIME_BETWEEN_ACTIVE_UPDATES)
     def update_active():
         """Update the active power usage."""
-        data.get_realtime()
+        realtime_data_wrapper.update_realtime()
 
     devices = []
     for sensor in config.get(CONF_MONITORED_CONDITIONS):
-        config_name, prod = sensor.rsplit('_', 1)
-        name = SENSOR_TYPES[config_name].name
-        sensor_type = SENSOR_TYPES[config_name].sensor_type
-        is_production = prod == PRODUCTION_NAME.lower()
-        if sensor_type == ACTIVE_TYPE:
-            update_call = update_active
+        if sensor != DEVICES_NAME.lower():
```

Comment:
I see the code `DEVICES_NAME.lower()` a few times referenced in your code. Please define a new constant with the name that you expect to be used.